### DISCUSSION TITLE
Minimal kernel versions for different operations:

### DISCUSSION THREAD

**Discussion by noteflakes** [axboe/liburing#1262] [General]
There are some ops for which the minimal kernel version is not indicated in the MAN pages. What is minimal kernel version for the following:

- io_uring_prep_waitid
- io_uring_prep_cmd_sock
- io_uring_prep_futex_wait / io_uring_prep_futex_wake

Thanks!

**Comment by axboe** [comment]
Ah indeed, let me go over those and get that added.

**Reply by axboe** [reply]
Done:

https://git.kernel.dk/cgit/liburing/commit/?id=5b46a7e5621d0431407b5c93a822246c118465b2
